### ISSUE TITLE
Can't do flutter get packages

### ISSUE THREAD

**Issue by drdgvhbh** [orthros/dart-epub#27]
Getting this when I do `flutter packages get`

```
The current Dart SDK version is 2.0.0-dev.63.0.flutter-4c9689c1d2.

Because e_reader depends on epub >=2.0.0 which requires SDK version >=2.0.0 <3.0.0, version solving failed.
```
Seems to be related/similar to the other issue in this project. I also noticed there was a recent [commit](https://github.com/orthros/dart-epub/commit/c57075a3d560731d17f209b28f9750899c3e14f7) that may be resolving this issue.

**Comment by drdgvhbh** [comment]
Now getting this

```
The current Dart SDK version is 2.0.0-dev.63.0.flutter-4c9689c1d2.

Because every version of flutter_test from sdk depends on quiver 0.29.0+1 and epub >=2.0.2 depends on quiver ^2.0.0, flutter_test from sdk is incompatible with epub >=2.0.2.
And because epub >=2.0.0 <2.0.2 requires SDK version >=2.0.0 <3.0.0, flutter_test from sdk is incompatible with epub >=2.0.0.
So, because e_reader depends on both epub ^2.0.0 and flutter_test any from sdk, version solving failed.
pub get failed (1)
```

**Comment by orthros** [comment]
Hello; Thanks for your patience. I think this should be fixed with the latest releases, and PR #42
